src/intelligence_layer/connectors/__init__.py

```diff
@@ -84,6 +84,5 @@
     QdrantInMemoryRetriever as QdrantInMemoryRetriever,
 )
 from .retrievers.qdrant_in_memory_retriever import RetrieverType as RetrieverType
-from .studio.studio import StudioClient as StudioClient
```

Comment:
do we still need this now that we got rid of the circular dependency? otherwise i would include it in here again

Reply:
Yes correct we now expose it